scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -101,7 +101,11 @@
       {% if case.individuals|length > 1 %}
         <span class="badge bg-primary">
           {% for item in case.individuals %}
-            <i class="fa fa-user"></i>
+            {% if case.track == "cancer" %}
+              <em class="fa fa-vial"></em>
```

Comment:
This is good enough; I find `<span></span>` better yet.

Reply:
Do you want me to replace em with span?

Reply:
Like this? 
```

{% if case.individuals|length > 1 %}
    {% for item in case.individuals %}
          {% if case.track == "cancer" %}
            <span class="fa fa-vial"></span>
          {% else %} <!-- rare disease case -->
            <span class="fa fa-user"></span>
          {% endif %}
     {% endfor %}
{% endif %}

```

<img width="240" alt="image" src="https://user-images.githubusercontent.com/28093618/173074047-0a50d6a7-6d7d-44c3-ae74-6574b884badb.png">

Reply:
Looks good! It could still be wrapped in the outer `span class=badge` regardless - either is fine really. But the whole business of `<i>` vs `<em>` is nicely resolved with `<span>` in this case, as there is no text or symbol to emphasize/italicise.

Reply:
In the end I've oped for a double, span, as you also suggested (I think!). Result:

<img width="172" alt="image" src="https://user-images.githubusercontent.com/28093618/173295623-ef619a76-2535-4831-9c49-5950c9c396d5.png">


<img width="199" alt="image" src="https://user-images.githubusercontent.com/28093618/173295671-5333baf4-77fc-48ec-b7d1-906fd548ff5b.png">